scripts/airgapped/get-all-images.sh

```diff
@@ -46,6 +46,9 @@ rm -rf resource-dispatcher
 # manually retrieve pipelines runner image to test pipelines
 IMAGES+=($(echo "charmedkubeflow/pipelines-runner:ckf-1.8"))
```

Comment:
It'd be great if this was not hardcoded here, I know it comes from a different PR, so nothing to work on. Maybe let's keep it in the back of our heads for improvements.

Reply:
I filed an issue to track the improvements for airgapped #933